babel.config.js

```diff
@@ -1,11 +1,14 @@
 module.exports = {
   presets: [
     [
-      "@vue/app",
+      "@vue/cli-plugin-babel/preset",
```

Comment:
what does this do?

Reply:
This is an array of babel plugins that are specifically for Vue. [Here ](https://babeljs.io/docs/en/presets)it explains what presets are (array of plugins so you don't need to configure them yourself)

We need this in order to have all the required plugins in our system to use ECMAScript 2020